### ISSUE TITLE
support Logon Actions

### ISSUE THREAD

**Issue by cshuig** [kingToolbox/WindTerm#312]
能支持类似 secureCRT中的 logon Actions 的功能吗。

**Comment by kingToolbox** [comment]
Yes, features such as `macros`, `triggers`, `events`, `notifications`, `plugins`, `scripts` and so on will be added in subsequent versions, which should meet your needs. I will update the progress here, please stay tuned, thank you.

**Comment by imjakey** [comment]
@kingToolbox  请问还有支持该功能的计划吗？
